application/frontend/web/index.php

```diff
@@ -14,20 +14,16 @@
         require(__DIR__ . '/../config/main.php')
     );
 } catch (\Exception $e) {
-    
-    // Log to syslog (Logentries).
-    openlog('id-broker', LOG_NDELAY | LOG_PERROR, LOG_USER);
-    syslog(LOG_CRIT, $e->getMessage());
-    closelog();
-    
+
     // Return error response code/message to HTTP request.
     header('Content-Type: application/json');
     http_response_code(500);
     $responseContent = json_encode([
         'name' => 'Internal Server Error',
         'message' => $e->getMessage(),
         'status' => 500,
-    ], JSON_PRETTY_PRINT);
+    ]);
+    fwrite(fopen('php://stderr', 'w'), $responseContent . PHP_EOL);
```

Comment:
Could this be simplified to use `STDERR` as [the example below](https://github.com/silinternational/idp-id-sync/pull/96/files#diff-6d8d0025a1ae5b985ef06b4d55d858f8R28) does, or is there something different about this context that prevents that usage?
```suggestion
    fwrite(STDERR, $responseContent . PHP_EOL);
```

Reply:
No, it comes from Yii, and it's not loaded at this point.